app/controllers/finders_controller.rb

```diff
@@ -141,13 +144,14 @@ def search_results
     search_query.search_results
   end
 
-  def suggestions
-    search_results.fetch("suggested_queries", []).map do |keywords|
-      {
-        keywords: keywords,
-        link: finder_url_builder.url(keywords: keywords),
-      }
-    end
+  def spelling_suggestion_presenter
+    suggested_queries = search_results.fetch("suggested_queries", [])
+    SpellingSuggestionPresenter.new(
+      suggested_queries,
+      finder_url_builder.url(keywords: suggested_queries.first),
+      #todo maybe back and make sure that search api returns only 1 thing and not and array with one item
```

Comment:
?

Reply:
search-api looks to always return an array, even as there is always one item. @laurentqro suggested a pr in search api to update that, but check with other devs first. 
that's why we left a comment, but i forgot to check 😊

Reply:
I guess it all depends on how many spelling suggestions we want to show users, and if so, which one. So it might actually be quite a bit of work.

Reply:
search-api is set up to return at most one suggestion, so I agree an array is a bit unnecessary.

But maybe once upon a time someone thought it was more convenient to have the value always returned, and have consumers check if the array is empty or not, than for consumers to check if the value exists or not.  Returning an array *does* mean we could return more suggestions in the future without much change

Reply:
I've updated the comment, happy to take suggestions for code to be updated to accept an array  and loop over, instead of doing `.first`